### PR TITLE
Updating Kiali to v0.5.0 in Helm Installer

### PR DESCRIPTION
**Pull request by gbaufake**
- As I was told in https://github.com/istio/istio/pull/6929, opening this PR to release-1.0.

Best Regards,
Guilherme Baufaker Rêgo

### CODE REVIEW

**Comment by gbaufake** [comment]
cc @gyliu513  @sdake  @rshriram @jmazzitelli

**Comment by sebastien-prudhomme** [comment]
Hi, it seems you forgot to add quotaspecs and quotaspecbindings in clusterole

File: install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml [hunk 1]
```diff
@@ -23,8 +23,6 @@ rules:
   - watch
 - apiGroups: ["config.istio.io"]
   resources:
-  - routerules
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml]: I think we still need `routerules` for backward compatibility.

File: install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml [hunk 2]
```diff
@@ -23,8 +23,6 @@ rules:
...
-  - destinationpolicies
```
Comment (gyliu513) [inline install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml]: Why delete `destinationpolicies`?

Comment (jmazzitelli) [reply install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml]: kiali at this point is not accessing routerules or destinationpolicies - we are using the new stuff, so we don't need these in our cluster roles.

File: install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml (hunk 1 above)
Comment (jmazzitelli) [reply install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml]: kiali 0.5.0 is not accessing routerules directly anymore - it is using the new destinationrules. So it does not need routerules specified in its clusterrole

File: install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml (hunk 2 above)
Comment (gyliu513) [reply install/kubernetes/helm/istio/charts/kiali/templates/clusterrole.yaml]: @jmazzitelli , @gbaufake does the Kiali document highlight this? With 0.5.0, Kiali will not able to view routerules?

Also why not show destinationpolicies? This is still used in istio.

**Comment by sebastien-prudhomme** [comment]
@gbaufake You should update requirements.yaml with the new version

Also can you add some way to override URL auto detected by Kiali?

For instance for the Jaeger one:

```diff
---istio-orig/charts/kiali/templates/deployment.yaml 2018-07-10 15:38:16.544494504 +0
200
+++ istio/charts/kiali/templates/deployment.yaml        2018-07-12 11:55:31.336640439 +0200
@@ -44,6 +44,14 @@
             secretKeyRef:
               name: kiali
               key: password
+{{- if .Values.dashboard.jaegerURL }}
+        - name: JAEGER_URL
+          value: {{ .Values.dashboard.jaegerURL }}
+{{- end }}
         volumeMounts:
         - name: kiali-configuration
           mountPath: "/kiali-configuration"
--- istio-orig/values.yaml    2018-07-12 09:53:19.860247767 +0200
+++ istio/values.yaml   2018-07-12 11:51:56.553894566 +0200
@@ -454,6 +454,9 @@
     username: admin
     password: admin
 
+    # Override the automatically detected Jaeger URL, usefull when Jaeger service has no ExternalIPs
+    # jaegerURL:
+
 certmanager:
   enabled: false
   replicaCount: 1
```

**Comment by gbaufake** [comment]
@sebastien-prudhomme 
Can you detail more your propose?

**Comment by gbaufake** [comment]
@gyliu513 I think it is fixed 09ff3ee!

@sebastien-prudhomme if you have better idea what to do with jager-url, you can create a PR, then I can review it.

**Comment by sebastien-prudhomme** [comment]
@gbaufake ok, waiting for this PR being merged

**Comment by costinm** [comment]
Lint seems unrelated.

**Pull request merged**